examples/custom_layout_manager/custom_layout.rs

```diff
@@ -0,0 +1,186 @@
+use crate::TOTAL_COLORS;
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+const N_GRID_COLUMNS: u32 = 4;
+
+mod imp {
```

Comment:
Why not move this into a separate file instead of an inline module? :)

Reply:
Because none of the examples in gtk4-rs does that and didn't want to have only one that's different than others. We should probably do that for all of them though but I wouldn't do it as part of this;

cc @Hofer-Julian as you have done something similar for gtk-rs

Reply:
I can adapt them.
Want to open an issue and assign it to me?

Reply:
Done, thanks Julian! :)